Mage/src/main/java/mage/abilities/dynamicvalue/LockedInDynamicValue.java

```diff
@@ -43,9 +45,19 @@ public LockedInDynamicValue copy() {
         return new LockedInDynamicValue(this, true);
     }
 
-    @Override
```

Comment:
why not override? (same question a few other places)

Reply:
good question. answer is "i missed that"